scala/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not a Maven expert. Would you mind explaining the purpose of this pom.xml? Thanks.